<launch>
  <arg name="UPDATE_RATE" default="1.0" />

  <node pkg="jsk_topic_tools" type="transform_merger" name="tf_buffer_server"
	output="screen">
    <param name="loop_hz" value="$(arg UPDATE_RATE)" />
    <remap from="/tf_merged" to="/tf_merged_update"/>
  </node>
</launch>
